frappe/patches/v13_0/validate_options_for_data_field.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
```

Comment:
frappe/frappe is MIT, also license.txt doesn't exist. Use a file header from one of the files that exist in the codebase maybe?